### ISSUE TITLE
MVC 4 Rotativa error "This program is blocked by group policy"

### ISSUE THREAD

**Issue by restorepro** [webgio/Rotativa#109]
Rotativa works great on localhost, but I get a error when the application is published to the server. The error is: "This program is blocked by group policy".  Stack Trace: [Win32Exception (0x80004005): This program is blocked by group policy. 
For more information, contact your system administrator]. 
Any idea on a fix for this? I googled this and found several people experienced this issue and I have not found a solution for it.  Thanks.

**Comment by restorepro** [comment]
Feeback on this issue: I spoke to Godaddy on 2 occasions and the permissions that would allow the exe file to work cannot be adjusted if you have a shared hosting account. They told me that I would need to upgrade to a dedicated hosting account for the permissions to be changed that would allow Rotativa's exe file to run.

**Comment by MuhammadQasimHunain** [comment]
declare full trust level in web config
<system.web>
<trust level="Full" />
</system.web>

**Comment by anbuArunakt503** [comment]
Muhammad - Already i have written this code in my application. even though i am getting this error

**Comment by arrauf** [comment]
I am also suffering from this error. ;(

**Comment by restorepro** [comment]
Hello arrauf! I think that my error was mainly because my application was being hosted in a shared environment. I upgraded to a dedicated server (godaddy) and it works fine. 
I also set the trust level to full in the web.config..
<system.web>
'Set trust to full here. <trust level="Full" />
</system.web>

Also, check this link to make sure that these files are in the Rotativa folder: 
My Rotativa folder also has these files:

    help-wkhtmltoimage.txt
    help-wkhtmltopdf.txt
    msvcp120.dll
    msvcp140.dll
    msvcr120.dll
    vcruntime140.dll
    wkhtmltoimage.exe
    wkhtmltopdf.exe 
    wkhtmltox.dll 

Nuget installed some of the files, I searched and found a couple on my machine and I installed a visual studio update to obtain the remaining 1 or 2 files. 
I think this the link for the update: https://www.microsoft.com/en-GB/download/details.aspx?id=40784

And last, I had to download the exe file in the link below and replace the wkhtmltopdf.exe file that the nuget package manager installed in the Rotativa folder... Rotativa would not work for me with the exe file that nuget installed for some reason...
Hope this helps!

https://drive.google.com/file/d/0BzI_u_O3vUM1aEppb3ZVZXNsc28/view
